student-library/src/main/java/tds/student/sql/data/OpportunityItem.java

```diff
@@ -58,6 +53,7 @@ public long getItemKey () {
     return _itemKey;
   }
 
+  @JsonProperty("ItemKey")
```

Comment:
So now the question becomes, why do all the private properties start with "_"? That's not very java-y.